docs/development/environment.md

```diff
@@ -8,10 +8,32 @@ Read our documentation about the [Developer Workflow](https://developers.matterm
 
 This plugin supports both Jira Server (self-hosted) and Jira Cloud instances. There can be slight differences in behavior between the two systems, so it's best to test with both systems individually when introducing new webhook logic, or adding a new Jira API call.
 
-To test your changes against a local instance of Jira Server, you need [Docker](https://docs.docker.com/install) installed, then you can use the `docker-compose.yml` file in this repository to create a Jira instance. Simply run `docker-compose up` in the directory of the repository, and a new Jira server should start up and be available at http://localhost:8080. It can take a few minutes to start up due to Jira Server's startup processes. If the container fails to start with `exit code 137`, you may need to increase the amount of RAM you are allowing docker to use.
+To test your changes against a Jira Cloud instance, we recommend starting a 14-day trial, if you don't have a Jira project to test against. More information can be found here: https://www.atlassian.com/software/jira/try.
 
 If you are contributing to a feature that requires multiple Jira instances to be installed, please enable [ServiceSettings.EnableDeveloper](https://docs.mattermost.com/configure/configuration-settings.html#enable-developer-mode) in your server's config in order to circumvent the Enterprise license requirement. 
 
-**Note:** You cannot use LOCALHOST to connect the Jira plugin to your server, you can use a proxy if needed.
+### Run a local instance of Jira server
 
-To test your changes against a Jira Cloud instance, we recommend starting a 14-day trial, if you don't have a Jira project to test against. More information can be found here: https://www.atlassian.com/software/jira/try.
+To test your changes against a local instance of Jira Server, you need [Docker](https://docs.docker.com/install) installed. 
+
+**Pre-requisite**
+As per the [sizing recommendations from Jira](https://confluence.atlassian.com/jirakb/jira-server-sizing-guide-975033809.html), it requires atleast a minimum memory of 8GB. Hence it is advised to increase the amount of resources allocated for your Docker to use. Here are the steps on how to do this using Docker Desktop:
+- Click on the Settings icon on the Docker Desktop
+- Navigate to Resources section
+- Ensure that the Memory is set to atleast to 8GB or more
```

Comment:
Man that's a lot of memory. Are we sure we need that much memory? I imagine Atlassian suggests that amount for production use specifically?

Reply:
Yes, Jira Server requires at least 8GM RAM bare minimum. I tried this with a reduced amount of resources, several times and the setup process hangs forever and is stuck at different places.